.github/workflows/dotnet.yml

```diff
@@ -1,4 +1,4 @@
-name: .Net
+name: .NET
```

Comment:
Is this relevant in this PR?

Reply:
Since I changed this file in the Friday session, it appeared in the Git Changes.

Reply:
I should have deleted before commit